libfreerdp/core/credssp_auth.c

```diff
@@ -111,6 +111,13 @@ BOOL credssp_auth_init(rdpCredsspAuth* auth, TCHAR* pkg_name, SecPkgContext_Bind
 	const rdpSettings* settings = auth->rdp_ctx->settings;
 	WINPR_ASSERT(settings);
 
+	char name[64] = { 0 };
+#if defined(UNICODE)
+	ConvertWCharToUtf8(pkg_name, name, ARRAYSIZE(name));
+#else
+	strncpy(name, pkg_name, ARRAYSIZE(name));
```

Comment:
This might produce a non-null terminated string for security packages with long names (>= 64)

Reply:
@mfleisz good catch, fixed.